keps/sig-node/2400-node-swap/README.md

```diff
@@ -401,14 +402,17 @@ For alpha:
   and further development efforts.
   - Focus should be on supported user stories as listed above.
 
-For beta:
+For alpha2:
 
 - Add e2e tests that exercise all available swap configurations via the CRI.
+- Verify MemoryPressure behavior with swap enabled and document any changes
+  for configuring eviction.
+- Verify new system-reserved settings for swap memory.
```

Comment:
Can you clarify a little bit on "Verify" here? Manual testing and figure out the work items for beta? Document the changes to guide the users for the alpha feature?